README.md

```diff
@@ -31,12 +31,12 @@ Wraps an action creator so that its return value is the payload of a Flux Standa
 Example:
 
 ```js
-let increment = createAction('INCREMENT', amount => amount);
+let noop = createAction('NOOP', amount => amount);
```

Comment:
Why is noop more suitable than increment for the action creator name? as the action created from this action creator is used to increment a counter?